documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2024,6 +2024,9 @@ In order to capture large object values and have them serialized in the change e
 |
 |A comma-separated list of RAC node host names or addresses.
 This field is required to enable Oracle RAC support.
+If a RAC node uses a port that differs from `database.port`, the entry can be specified as `ip:port` to provide the node's custom port. +
+ +
+When using `database.url`, a `database.port` property is not required but all RAC nodes must then be given using the format `ip:port`.
```

Comment:
When the xref:<oracle-property-database-url[`database.url`] property is specified for the connector, a xref:<oracle-property-database-port[`database.port`] property is not required.
However, for each RAC node that you list for this property, you must use the following format to specify the RAC node address: `__<ip>__:__<port>__`

Reply:
I'm not 100% sure that I understand the intent of the preceding passage, so if I'm totally off-base, let me know.

Reply:
The goal here with these two passages was to convey the rules I outlined above:

* If a `database.url` property is used by the connector configuration, a `database.port` isn't required.
* When specifying `rac.nodes`, the RAC node's port can be defaulted by specifying `database.port`.  If a RAC node uses a different port or if the user wishes to specify the ip/port here and exclude specifying `database.port`, they can provide the entry in `rac.nodes` using the format of `__<ip>__:__<port>__`. specified using the format of `__<ip>__:__<port>__`.  

Does that help clarify how this should be worded?

Reply:
> Does that help clarify how this should be worded?

Yeah, so how about something like this:

|A comma-separated list of RAC node host names or addresses.
This property is required to enable Oracle RAC support.
You can use several methods to specify the list of RAC nodes. +
 +
You can specify a value for `xref:<oracle-property-database-port[`database.port`], and use the specified port value for each address in the `rac.nodes` list. For example:

----
database.port=1521
rac.nodes=192.168.1.100,192.168.1.101,192.168.1.102
----

If you specify a value for `database.port`, you can override the default port for one or more entries in the `rac.nodes` list. That is, the list can include some entries that use the default `database.port` value, and some entries that define their own unique port values. For example, 

----
database.port=1521
rac.nodes=192.168.1.100,192.168.1.101,192.168.1.102,192.168.1.103:1522
----

If instead of defining a value for `database.port` you supply a raw JDBC URL for the database by using the xref:<oracle-property-database-url[`database.url`] property, you must explicitly specify the port values for each RAC node in the list.

In the preceding, I mixed the 2nd and 3rd examples in our chat. If you think that it's not clear enough, I can break those out into separate examples.

Reply:
Thanks for the feedback, but won't this be a bit much for the Notes column?